aws_lambda_powertools/utilities/data_classes/vpc_lattice.py

```diff
@@ -0,0 +1,36 @@
+from typing import Dict
+
+from aws_lambda_powertools.utilities.data_classes.common import (
+    DictWrapper,
+)
+
+class VPCLatticeEvent(DictWrapper):
```

Comment:
@heitorlessa  I was looking at possibly using that as your suggested.  However, at least one attribute in the BaseProxyEvent would not apply to any possible VPCLatticeEvent that would be inherited from that.  

I am leaning towards having a new event proxy similar to BaseProxyEvent and pull in the base64 decode and json decode and headers options since I think you are correct that people would eventually ask for those.